opencsp/common/lib/csp/FacetEnsemble.py

```diff
@@ -219,6 +205,9 @@ def set_facet_positions(self, positions: Pxyz):
             facet._self_to_parent_transform = TransformXYZ.from_V(pos)
 
     def set_facet_canting(self, canting_rotations: list[Rotation]):
+        """Sets facet canting relative to ensemble.
```

Comment:
Does this apply to a single facet or multiple facets? In other words, should this be changed to "set_facet_cantings"?

Reply:
Done.